TimerTriggerAttachmentsVerify/index.ts

```diff
@@ -0,0 +1,27 @@
+import { AzureFunction, Context } from "@azure/functions";
+import * as df from "durable-functions";
+
+const timerTrigger: AzureFunction = async (
+  context: Context,
+  // tslint:disable-next-line: no-any
+  myTimer: any
+): Promise<void> => {
+  const timeStamp = new Date().toISOString();
+  const client = df.getClient(context);
+
+  if (myTimer.IsPastDue) {
+    // help at start time
+    context.log("Timer function is running late!");
+  }
+
+  // name of the orchestrator
+  const functionName = "VerifyAttachmentsOrchestrator";
+  const instanceId = "timerVerify";
+  context.log(
+    `Started timer for launching verify orchestrator with ID = '${instanceId}'.`
+  );
+  await client.startNew(functionName, instanceId);

```

Comment:
it is not clear why we have to create an orchestrator that calls an activity function. can't we just call
`U.verifyAllAttachments().map(taskEmails => taskEmails.run());` here ?

Reply:
I thought about it or also about calling directly an activity instead. But at some point I read this below from ms documentation and I convinced myself that is better to use the orchestrator capabilities. Also orchestrator provides their status API. 

Instead of using infinite loops, orchestrator functions reset their state by calling the ContinueAsNew (.NET) or continueAsNew (JavaScript) method of the orchestration trigger binding. This method takes a single JSON-serializable parameter, which becomes the new input for the next orchestrator function generation.

When ContinueAsNew is called, the instance enqueues a message to itself before it exits. The message restarts the instance with the new input value. The same instance ID is kept, but the orchestrator function's history is effectively truncated.